content/en/project/status/2023MedleyAnnualReport.md

```diff
@@ -0,0 +1,91 @@
+# 2023 Medley Interlisp Project Annual Report
```

Comment:
Replace with correct header in for Hugo:

```
---
title: 2023 Medley Interlisp Annual Report
weight: 1
type: docs
---
```

You will also want to change the weight on the 2022 Annual Report to another value, `9` is probably good to align it right before the 2021 Annual Report.